http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/DiskRoutes.scala

```diff
@@ -194,9 +197,10 @@ object DiskRoutes {
     } yield UpdateDiskRequest(l.getOrElse(Map.empty), us)
   }
 
-  implicit val getDiskResponseEncoder: Encoder[GetPersistentDiskResponse] = Encoder.forProduct12(
+  implicit val getDiskResponseEncoder: Encoder[GetPersistentDiskResponse] = Encoder.forProduct13(
     "id",
     "googleProject",
+    "cloudContext",
```

Comment:
I think there's a bug here based on how the signature of `GetPersistentDiskResponse` changes here (the number of fields don't seem to change?)

https://github.com/DataBiosphere/leonardo/pull/2339/files#diff-69288d2ed640980de319fedf8f45eb9f03c794e90cda4239dac24cb1ebe0e9d0R40